ibm_catalog.json

```diff
@@ -0,0 +1,460 @@
+                    "value": 30
+                  },
+                  {
+                    "displayname": "60",
+                    "value": 60
+                  },
+                  {
+                    "displayname": "90",
+                    "value": 90
+                  }
+                ]
+              },
+              {
+                "key": "logs_cos_bucket_name"
+              },
+              {
+                "key": "metrics_cos_bucket_name"
+              },
+              {
+                "key": "cloud_logs_cos_buckets_class",
+                "options": [
+                  {
+                    "displayname": "Standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "Vault",
+                    "value": "vault"
+                  },
+                  {
+                    "displayname": "Cold",
+                    "value": "cold"
+                  },
+                  {
+                    "displayname": "Smart",
+                    "value": "smart"
+                  },
+                  {
+                    "displayname": "One-Rate Active",
+                    "value": "onerate_active"
+                  }
+                ]
+              },
+              {
+                "key": "skip_cos_kms_iam_auth_policy"
+              },
+              {
+                "key": "existing_kms_key_crn"
+              },
+              {
+                "key": "cloud_log_storage_key_ring"
+              },
+              {
+                "key": "cloud_log_storage_key"
+              },
+              {
+                "key": "ibmcloud_kms_api_key"
+              },
+              {
+                "key": "existing_event_notifications_instances"
+              },
+              {
+                "key": "skip_logs_routing_auth_policy"
+              },
+              {
+                "key": "logs_routing_tenant_regions"
+              },
+              {
+                "key": "logs_policies"
+              }
+            ],
+            "iam_permissions": [
+              {
+                "service_name": "logs",
+                "role_crns": [
+                    "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ]
+              },
+              {
+                "service_name": "cloud-object-storage",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ]
+              },
+              {
+                "service_name": "kms",
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                  "crn:v1:bluemix:public:iam::::role:Editor"
+                ]
+              }
+            ],
+            "architecture": {
+              "descriptions": "This architecture supports creating and configuring a Cloud Logs instance.",
+              "features": [
```

Comment:
Can you list the features like the way its done in SCC DA so users can see difference between Fully configurable and security enforced ?